CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ else()
 endif()
 
 option(CODE_COVERAGE "Enable coverage reporting" OFF)
-option(UNIT_TESTING "Enable unit testing" OFF)
+option(UNIT_TESTING "Enable unit testing" ON)
```

Comment:
I think we should keep the default value `OFF`. One can set this value `UNIT_TESTING` as
```
cmake .. -DUNIT_TESTING=ON
```

Reply:
Changed to OFF